src/sentry/uptime/consumers/results_consumer.py

```diff
@@ -407,6 +429,58 @@ def has_reached_status_threshold(
             )
         return result
 
+    def _produce_snuba_uptime_result(
+        self, project_subscription: ProjectUptimeSubscription, result: CheckResult
+    ) -> None:
+        """
+        Produces a message to Snuba's Kafka topic for uptime check results.
+
+        Args:
+            project_subscription: The project subscription associated with the result
+            result: The check result to be sent to Snuba
+        """
+        try:
+            project = project_subscription.project
+            retention_days = (
+                quotas.backend.get_event_retention(organization=project.organization) or 90
+            )
+
+            snuba_message: SnubaUptimeResult = {
```

Comment:
Should we just `deepcopy` and then add more fields into the result? Or are there extra fields that you're also removing?

Reply:
i like this in order to reduce the lines of code needed, but mypy does not. i'd prefer not to have to use cast, so i think keeping it as is is best? lmk your thoughts though

Reply:
Ahh, since there's no direct inheritance I guess it fails here... not a huge deal imo